src/components/PromiseDetail/PromiseStatusTag.svelte

```diff
@@ -3,15 +3,17 @@
 	import { twMerge } from 'tailwind-merge';
 
 	export let status: PromiseStatus;
+	export let isLarge = true;
```

Comment:
I'm sorry for not getting back to you sooner. Everything is working as expected! I suggest using optional boolean props' default value as `false`, because:
- We don't make it large when omitting it from the component like `<PromiseStatusTag>`.
- Then when we make it large, we use `<PromiseStatusTag isLarge>` as Svelte treats this as true without putting `={true}`.

I think this feels more natural. The only problem with this component is that it makes breaking changes, but I think finding all in the codebase and replacing them is worth doing. (Also making the API like other Tag components)

Reply:
Good point na. Just changed it. Thank youu.